Libraries/Components/TextInput/TextInput.js

```diff
@@ -1392,25 +1402,38 @@ function InternalTextInput(props: Props): React.Node {
   // so omitting onBlur and onFocus pressability handlers here.
   const {onBlur, onFocus, ...eventHandlers} = usePressability(config) || {};
 
-  const _accessibilityState = {
-    busy: props['aria-busy'] ?? props.accessibilityState?.busy,
-    checked: props['aria-checked'] ?? props.accessibilityState?.checked,
-    disabled: props['aria-disabled'] ?? props.accessibilityState?.disabled,
-    expanded: props['aria-expanded'] ?? props.accessibilityState?.expanded,
-    selected: props['aria-selected'] ?? props.accessibilityState?.selected,
-  };
+  let _accessibilityState;
```

Comment:
nit: we should chose a different way than underscore to avoid shadowing. The ESLint rules special case variables starting with underscore as allowed to be possibly unused, and it is commonly used in the codebase for instance-like variables.